python/lsst/ts/standardscripts/maintel/laser_tracker/measure.py

```diff
@@ -0,0 +1,172 @@
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["Measure", "AlignComponent"]
+
+import asyncio
+import enum
+
+import yaml
+from lsst.ts import salobj
+from lsst.ts.idl.enums.LaserTracker import LaserStatus
+from lsst.ts.observatory.control import RemoteGroup
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+from lsst.ts.observatory.control.remote_group import Usages
+
+
+# TODO (DM-38880) - Subsitute by class in idl.enum when available
+class AlignComponent(enum.IntEnum):
+    M2 = 1
+    Camera = 3
+
+
+class Measure(salobj.BaseScript):
+    """Measure component using laser tracker.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+    - "Starting alignment procedure.": Starting alignment with laser tracker.
+    - "M2 Hexapod aligned with laser tracker.": M2 aligned.
+    - "Camera Hexapod aligned with laser tracker.": Camera measured.
+    """
+
+    def __init__(self, index: int, add_remotes: bool = True):
+        super().__init__(index, descr="Measure MTCS components with laser tracker.")
+
+        self.laser_tracker = RemoteGroup(
+            domain=self.domain,
+            components=["LaserTracker:1"],
+            intended_usage=None if add_remotes else Usages.DryTest,
+            log=self.log,
+        )
+
+        self.mtcs = MTCS(
+            domain=self.domain,
+            intended_usage=None if add_remotes else MTCSUsages.DryTest,
+            log=self.log,
+        )
+
+        self.timeout_measure = 120
+        self.timeout_std = 60
+        self.timeout_short = 5
+
+        self.target = None
+
+    @classmethod
+    def get_schema(cls):
+        url = "https://github.com/lsst-ts/"
+        path = (
+            "ts_externalscripts/blob/main/python/lsst/ts/standardscripts/"
+            "maintel/laser_tracker/measure.py"
+        )
+        schema_yaml = f"""
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: {url}{path}
+        title: MaintelLaserTrackerAlign v1
+        description: Configuration for Maintel laser tracker measurement SAL Script.
+        type: object
+        properties:
+            target:
+                description: >-
+                    Target to measure. Options are:
```

Comment:
The "Options" are already self documented in the enum. I think you can remove this from the description